content/docs/misc/oadp/rosa-sts/_index.md

```diff
@@ -226,13 +228,6 @@ and restore process, but it should be noted as there are issues with it.
          - aws
        restic:
          enable: false
-     volumeSnapshots:
```

Comment:
We might need this right to ensure the enableSharedConfig is true, or am I wrong?

Reply:
If someone can demo to wes VSL works in this fashion he'd add it back in later.

Reply:
I don't think @weshayutin followed mobb docs exactly tho.. he didn't have config set like this.
```
     - velero:
         config:
           credentialsFile: /tmp/credentials/openshift-adp/cloud-credentials-credentials
           enableSharedConfig: "true"
           region: ${REGION}
         provider: aws
```

Reply:
@shawn-hurley happy to work w/ you on snapLocation, didn't work for me.